okhttp/src/main/java/okhttp3/internal/platform/Platform.java

```diff
@@ -133,6 +133,20 @@ public boolean isCleartextTrafficPermitted(String hostname) {
     return true;
   }
 
+  public Object captureAllocationSite() {
+    if (!logger.isLoggable(Level.FINE)) return null;
+    return new Throwable("response.body().close()");
```

Comment:
Get rid of `Thread.currentThread().getStackTrace()`. That's probably the worst way of getting that information.